cmd/ctr/utils_unix.go

```diff
@@ -21,8 +21,8 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
-func prepareStdio(stdin, stdout, stderr string, console bool) (*sync.WaitGroup, error) {
-	var wg sync.WaitGroup
+func prepareStdio(stdin, stdout, stderr string, console bool) (wg *sync.WaitGroup, err error) {
+	wg = &sync.WaitGroup{}
```

Comment:
Maybe predeclare `f` too to avoid the `:=` in the first call to `OpenFifo`?

`wg`, `f` and `ctx` could be put in a single `var` block.